contrib/features/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.core.contrib</groupId>
+    <artifactId>org.openhab.core.contrib.reactor</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.openhab.core.contrib.features</groupId>
+  <artifactId>org.openhab.core.contrib.reactor.features</artifactId>
+  <packaging>pom</packaging>
+
+  <name>openHAB Core :: Contrib :: Features</name>
+
+  <modules>
+    <module>karaf</module>
```

Comment:
As we dropped p2 features we could discuss if we needs a separate `karaf` directory at all... We could define features as karaf features, use karaf as a directory instead of feature (so instead of `features/karaf` simply `karaf`), ...

@kaikreuzer already mentioned (and I agree) if Karaf features is the only member of features, another subdirectory doesn't make sense....